packager/packager.cc

```diff
@@ -930,14 +938,15 @@ vector<ast::ParsedFile> Packager::run(core::GlobalState &gs, WorkerPool &workers
     return files;
 }
 
-vector<ast::ParsedFile> Packager::runIncremental(core::GlobalState &gs, vector<ast::ParsedFile> files) {
+vector<ast::ParsedFile> Packager::runIncremental(core::GlobalState &gs, vector<ast::ParsedFile> files,
+                                                 vector<std::string> extraPackageFilesDirectoryPrefixes) {
```

Comment:
Instead of copying `extraPackageFilesDirectoryPrefixes` on a call to this function, can we pass it as `const &`?